File: app/src/main/java/wannabit/io/cosmostaion/dao/Price.java
package wannabit.io.cosmostaion.dao;

import com.google.gson.annotations.SerializedName;

public class Price {
    @SerializedName("denom")
    public String denom;

    @SerializedName("current_price")
    public Double current_price;

    @SerializedName("daily_price_change_in_percent")
    public Double daily_price_change_in_percentage;

}

File: app/src/main/java/wannabit/io/cosmostaion/data/model/res/Price.kt
package wannabit.io.cosmostaion.data.model.res

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class Price(
    val denom: String?,
    val coinGeckoId: String?,
    val current_price: Double?,
    val daily_price_change_in_percent: Double?,
)
